### ISSUE TITLE
Shaders sometimes apply to previous states

### ISSUE THREAD

**Issue by vanjac** [ephtracy/ephtracy.github.io#149]
It seems like whenever I run a shader, it saves the state of the voxels for that shader. When I run the shader in the future it restores the voxels to that previous state. I'm not sure if that's exactly what's happening.

Here's a way to replicate it with the [sand](https://github.com/lachlanmcdonald/magicavoxel-shaders/blob/master/shader/sand.txt) shader:
- Delete all voxels
- Build a few layers of voxels on the ground
- Run `xs sand`
- Undo the previous two steps so everything is deleted again
- Run `xs sand` again. The previous state will be restored

Basically all shaders work once, but have problems if you run them multiple times. Using `xs -n` still works, but again only once.

(I hope this is the right place for bug reports?)

**Comment by ephtracy** [comment]
Hi, I have uploaded a new version to fix this issue. Thanks for the report.

**Comment by vanjac** [comment]
That fixed it, thank you!
